app/logic/events.py

```diff
@@ -62,6 +62,18 @@ def deleteEvent(eventId):
 
         event.delete_instance(recursive = True, delete_nullable = True)
 
+def deleteAllRecurringEvents(eventId):
```

Comment:
This function name is not entirely accurate, and will not be accurate when the requested changes get implemented. Is there a name the could better fit what the function is doing?

Reply:
Changed to deleteRecurringSeries()